edgequest.py

```diff
@@ -3205,6 +3203,7 @@ def toggle_fullscreen():
     ''' Toggle fullscreen mode in libtcod '''
 
     libtcod.console_set_fullscreen(not libtcod.console_is_fullscreen())
+    print 'Toggled fullscreen mode'
```

Comment:
I'm not really feeling this, considering the console shouldn't be utilized unless if there's a serious problem when testing